changelog/_unreleased/2025-02-11-remove-laguage-switch-from-document-settings.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Remove language switch from settings > documents
+issue: NEXT-40200
+---
+# Administration
+* Removed language switch from Settings > Documents
+  - Use different documents for different SalesChannel if required
```

Comment:
I would like to see the reason, why we remove it as well: because it didn't do anything on these pages